src/Inventory.js

```diff
@@ -7,6 +7,7 @@ class Inventory extends Map {
   addItem(item) {
     this.set(item.uuid, item);
     item.isHeld = true;
+    item.container = this;
```

Comment:
Any suggestions on how to resolve this? I thought the inventory reference would be okay since it would work for both containers and for items in a player's inventory but if nothing else it should have a different name.